### PR TITLE
testsuite: indentation fix in runtests.in

### PR DESCRIPTION
**Pull request by hzhou**
## Pull Request Description

*Fix the left-over bad indentation.

## Author Checklist
* [x] Reference appropriate issues (with "Fixes" or "See" as appropriate)
* [ ] Passes tests (included warning check)
* [x] Confirm whitespace/style checkers are happy (or has a good reason for being bad)
* [x] Commits are self-contained and do not do two things at once
* [x] Remove xfail from the test suite when fixing a test
* [x] Commit message is of the form: `module: short description` and follows [good practice](https://chris.beams.io/posts/git-commit/)
* [x] Add comments such that someone without knowledge of the code could understand
* [x] Add Devel Docs in the `doc/` directory for any new code design

### CODE REVIEW

**Comment by hzhou** [comment]
test:jenkins/ch4/ucx

File: test/mpi/runtests.in [hunk 1]
```diff
@@ -437,229 +436,230 @@ sub RunList {
             # just skip, do not complain missing unless it is "testlist"
             next;
         }
+        print "Looking in $curdir/$_f\n" if $debug;
```
Comment (raffenet) [inline test/mpi/runtests.in]: nitpick: there are 2 changes here, right? We are fixing the indentation and also moving a print to after a potential loop iteration skip.

Comment (hzhou) [reply test/mpi/runtests.in]: Yes, I'll split it.

**Comment by hzhou** [comment]
test:jenkins/ch3/tcp

**Comment by hzhou** [comment]
test:jenkins/ch3/tcp

**Comment by hzhou** [comment]
rebased to master, run tests one more time:
test:jenkins/ch4/most

**Pull request merged**